.github/workflows/python-package.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Python package
```

Comment:
Could this title be more descriptive about what's happening in this workflow? Maybe "Run tests and checks"?